doc/manual/rl-next/nix-flake-show-description.md

```diff
@@ -0,0 +1,25 @@
+---
+synopsis: Show package descriptions with `nix flake show`
+issues: [10977]
+prs: [10980]
+---
+
+`nix flake show` will now display a package's `meta.description` if it exists. If the description does not fit in the terminal it will be truncated to fit the terminal width.
```

Comment:
Isn't there also a limit of 80 chars?

https://github.com/NixOS/nix/pull/10980/files#diff-98db1e334292594945051880fac0aecb52c9206898bd3cbc79730d647b260824R1307